lib/m2ts/m2ts.js

```diff
@@ -81,6 +84,9 @@ TransportPacketStream = function() {
     }
   };
 
+  /**
+   * Passes identified M2TS packets to the TransportParseStream to be parsed
```

Comment:
Same re: TransportParseStream

Reply:
I think the comment for NalByteStream would be applicable here too since TransportParseStream is contained within TransportPacketStream 🤔